### PR TITLE
✨ rubick 008 release

### PR DESCRIPTION
**Pull request by roiLeo**
New rubick release `https://squid.subsquid.io/rubick/v/008/graphql`

ref kodadot/rubick#163 & kodadot/packages#83

### CODE REVIEW

**Review by Jarsen136 — APPROVED** [review]
lgtm

**Comment by yangwao** [comment]
🙅‍♂️


image

**Comment by roiLeo** [comment]
Oh no...
https://deploy-preview-4610--koda-
Capture d’écran 2023-01-05 à 12 46 07 PM
nuxt.netlify.app/series-insight

**Comment by yangwao** [comment]
Waiting for https://github.com/kodadot/rubick/pull/164 to be deployed

**Comment by roiLeo** [comment]
> Waiting for
> 
>     * [🔧 metadata gateway rubick#164](https://github.com/kodadot/rubick/pull/164) to be deployed
>       May back check in ~ 10 minutes?

Did you made a release?

**Comment by yangwao** [comment]
probably now yes 😏

**Comment by roiLeo** [comment]
Something wrong with 008 cache is not updating, can't query Series, Spotlight & Collector_whale

I'll investigate 🥷

**Comment by yangwao** [comment]
just called rockstar next to me @vikiival

**Comment by vikiival** [comment]
Testing it 
Tracked in 

- https://github.com/kodadot/rubick/issues/169

File: queries/config/.graphqlconfig [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "extensions": {
     "endpoints": {
       "Default GraphQL Endpoint": {
-        "url": "https://squid.subsquid.io/rubick/v/007/graphql",
+        "url": "https://squid.subsquid.io/rubick/v/008/graphql",
```
Comment (Jarsen136) [inline queries/config/.graphqlconfig:7]: How about using `URLS.koda.rubick` variable to replace this string?

Comment (roiLeo) [reply queries/config/.graphqlconfig:7]: Well this is a json file, I couldn't find any documentation about moving this config file to ts (maybe https://the-guild.dev/graphql/config/docs ?)

Comment (Jarsen136) [reply queries/config/.graphqlconfig:7]: Aright. Let's keep this so.

**Comment by vikiival** [comment]
Building rubick 008 
- kodadot/rubick@80bae9e77ebbfffe26ba85fa1f6c5d464662fd54

```
🦑 Releasing the squid at git@github.com:kodadot/rubick.git#80bae9e77ebbfffe26ba85fa1f6c5d464662fd54
◷ You may now detach from the build process by pressing Ctrl + C. The Squid deployment will continue uninterrupted.
◷ The new squid will be available as soon as the deployment is complete.
```

**Comment by roiLeo** [comment]
> [You can check the progress here](https://squiki.kodadot.workers.dev/status/rubick)

Something went wrong?

**Comment by vikiival** [comment]
> Something went wrong?

Looks like

Screenshot 2023-01-10 at 13 42 23

**Comment by vikiival** [comment]
Asked in the SubSquid

**Comment by vikiival** [comment]
https://github.com/kodadot/rubick/issues/174

**Comment by roiLeo** [comment]
deploymentUrl | : | https://squid.subsquid.io/rubick/graphql
-- | -- | --


🤔 new endpoint?

**Review by vikiival — APPROVED** [review]

**Pull request merged**